controller/JobRoleController.js

```diff
@@ -0,0 +1,65 @@
+"use strict";
```

Comment:
this is the compiled JS version of your TS file. these shouldn't be committed and should be generated in a directory such as /dist which can be ignored by git